python/PEDvPUDplot.py

```diff
@@ -2,104 +2,166 @@
+    ::param seq1Count::   Var to keep count of 1st sequence ID in pair.
+    ::param seq2Count::   Var to keep count of 2nd sequence ID in pair.
+    ::param eachseqpair:: Tuple of sequence IDs
+    ::param PUDS::        Dictionary of pairwise sequence keys and uncorrected distance values, excluding pairwise comparisons of sequence with itself.
+    """
     SDTid = getSDTid(filepath)
-    SDTdist = getSDTdist(filepath) # distance values
-    seqpairs = [] # tuple keys
-    i = 1
-    while i<len(SDTid):
-        j = 0
-        while j < i:
-            eachseqpair = (SDTid[i], SDTid[j])
+    SDTdist = getSDTdist(filepath) 
+    seqpairs = [] 
+    seq1Count = 1
+    while seq1Count<len(SDTid):
+        seq2Count = 0
+        while seq2Count < seq1Count:
+            eachseqpair = (SDTid[seq1Count], SDTid[seq2Count])
             seqpairs.append(eachseqpair)
-            j += 1
-        i += 1
+            seq2Count += 1
+        seq1Count += 1
     PUDS = dict(itertools.izip(seqpairs, SDTdist))
     return PUDS
 
-# function to get a list of lists comprising sequence ID and TreePuzzle distances
 def getTPZiddist(filepath2):
+    """
+    Reads Tree-Puzzle 5.2 maximum likelihood distance file (outdist.txt) and returns list of lists comprising sequence id (first element) and all pairwise distances.
+    
+    ::param filepath2::      Path to Tree-Puzzle file (input).
+    ::param tpzfile::        Input file.
+    ::param numseq::         Number of sequences in file.
+    ::param line::           Line from input file.
+    ::param eachline2::      List of elements from param line.
+    ::param item::           Each element in param eachline2.
+    ::param eachline2clean:: List of elements from param line, minus empty strings.
+    ::param seqid2element:: List of sequence ID string (first element) and pairwise distances for that sequence (subsequent elements).
+    ::param seqid2::        List of param seqid2element.    
+    """
     with open(filepath2) as tpzfile:
         numseq = tpzfile.readline()
         numseq = numseq.strip('\n')
         numseq = int(numseq)
         seqid2 = []
         seqid2element = []
-        tpzbyid = []
-        for lines in tpzfile:
-            eachline2 = lines.split(' ')
+        eachline2clean = []
+        for line in tpzfile:
+            eachline2 = line.split(' ')
             for item in eachline2:
                 item = item.strip('\n')
                 if item != '':
-                    tpzbyid.append(item)
+                    eachline2clean.append(item)
         while len(seqid2) < numseq:
-            seqid2element = tpzbyid[:numseq+1]
+            seqid2element = eachline2clean[:numseq+1] 
             seqid2.append(seqid2element)
-            del tpzbyid[:numseq+1]
+            del eachline2clean[:numseq+1]
     return seqid2
 
-# function to get pairwise evolutionary distances from TreePuzzle
-# key (tuples of sequence IDs): value (TreePuzzle identities))
 def getPED(filepath2):
```

Comment:
+ Will change.
+ To note which of the two input files is supposed to be called.